services/funfuzz/recipes/set_mercurial_config.sh

```diff
@@ -8,6 +8,7 @@ pushd "$HOME"
 # Populate Mercurial settings.
 cat << EOF > .hgrc
 [ui]
+username = gkw
```

Comment:
Is it right to use your own username, or should there be a generic account here?

Reply:
I can change it to `funfuzz`.